pages/09.Channels/02.Emails/01.Managing emails/docs.en.md

```diff
@@ -81,6 +81,33 @@ Your date will displayed as human reading format taken from  Configuration > Sys
 - Default format for date only  
 - Default Time Only Format 
 
+### Today token
+
+Token `{today}` display actual date time to the email. By default, you can use predefined formats after `|` symbol:
+
+- `{today}` same as `{today|datetime}`
+- `{today|date}`
+- `{today|time}`
+
+Your date will display as human reading format taken from  Configuration > System Settings
+
+- Default format for date only
+- Default Time Only Format
+
+#### Custom format 
+
+Returns the formatted date string based on PHP https://www.php.net/manual/en/datetime.format.php
```

Comment:
```suggestion
Returns the formatted date string based on PHP datetime format supplied - see [documentation][php-datetime]
```

Please add a macro [php-datetime] at the bottom of the page linking to https://www.php.net/manual/en/datetime.format.php